spec/text_handler_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'text_handler'
+
+describe TextHandler do
+  it 'sends a confirmation text to a specified phone number' do
+    text_client = double(:text_client)
+    allow(text_client).to receive_message_chain(:new, :messages, :create)
```

Comment:
Could this unit test be stronger by expecting `create()` to be called with specific arguments?